pkg/apis/kudo/v1beta1/operatorversion_types.go

```diff
@@ -120,9 +120,9 @@ type Task struct {
 // with the same json names as it would become ambiguous for the default parser. We might revisit this approach in the
 // future should this become an issue.
 type TaskSpec struct {
-	ResourceTaskSpec
-	DummyTaskSpec
-	PipeTaskSpec
+	ResourceTaskSpec `json:""`
```

Comment:
Let's use the `inline` specifier, like it's done for similar fields in this file.
```suggestion
	ResourceTaskSpec `json:",inline"`
```